src/unibit.js

```diff
@@ -240,6 +240,7 @@ module.exports = class Unibit {
         context.getPage = this.getPage.bind(this, context);
         context.getPages = this.getPages.bind(this, context);
         context.getData = this.getData.bind(this, context);
+        context.getPageByReference = this.getPageByReference.bind(this, context);
```

Comment:
I am having second thoughts about the name of this function.
Should it really be `getPageByReference`? Maybe a better name would be `getPageByFilePath` because it reflects what this function does.